core/storage/user/gae_models.py

```diff
@@ -343,7 +343,7 @@ class ExplorationUserDataModel(base_models.BaseModel):
     # The exploration version that this change list applied to.
     draft_change_list_exp_version = ndb.IntegerProperty(default=None)
     #The exploration draft version of this change list.
-    draft_change_list_id = ndb.IntegerProperty(default=None)
+    draft_change_list_id = ndb.IntegerProperty(default=0)
```

Comment:
If I understand correctly, this can be 0 if the draft is None or if the draft contains something (the latter case is for users who already have drafts before the draft ID was introduced). That is your intention, right? If so, could you please add a note to that effect here? Otherwise it might seem weird to devs that we have a draft ID of 0 corresponding to a draft of None.